src/wp-admin/includes/user.php

```diff
@@ -141,9 +137,14 @@ function edit_user( $user_id = 0 ) {
 
 	$errors = new WP_Error();
 
-	/* checking that username has been typed */
-	if ( '' === $user->user_login ) {
-		$errors->add( 'user_login', __( '<strong>Error:</strong> Please enter a username.' ) );
+	/* Validate the user_login when not updating the user */
```

Comment:
```suggestion
	// Validate the user_login when not updating the user.
```

Reply:
Single line comments should be `//`, per https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#5-inline-comments